### PR TITLE
User keymap for AKB/Raine

### PR DESCRIPTION
**Pull request by plikki**
## Description

This request is for a user made keymap for the Raine keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/akb/raine/keymaps/MehAdviceGuy/keymap.c [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define ALT_TAB LALT(KC_TAB)
```
Comment (drashna) [inline keyboards/akb/raine/keymaps/MehAdviceGuy/keymap.c]: This isn't used, and can be removed. 

```suggestion
```

Comment (plikki) [reply keyboards/akb/raine/keymaps/MehAdviceGuy/keymap.c]: Hello Drashna,
Thank you for pointing this out to me! 
I have removed this in commit "7d9d5d2184ff013c51213b68ef1893c0af0d420a"

Best regards

**Comment by plikki** [comment]
Hello! 
Sorry for the removing and adding files, but i did not seem to be able to rename files through github desktop, nor via the command line.

Thank you for letting me know that the naming was wrong, i must have missed that somewhere, they have been updated to be lowercase

Best Regards

**Comment by drashna** [comment]
Renaming can be tricky.  For command line, if you name it to something different, commit, and then name it back, but lowercase and commit, that will pick up the change.

**Pull request merged**